types/params_test.go

```diff
@@ -24,116 +24,115 @@ func TestConsensusParamsValidation(t *testing.T) {
 		params ConsensusParams
 		valid  bool
 	}{
-		// test block params
+		// valid params
 		{
-			name: "normal values",
+			name: "minimal setup",
+			params: makeParams(makeParamsArgs{
+				blockBytes:  1,
+				evidenceAge: 2,
+			}),
+			valid: true,
+		},
+		{
```

Comment:
Make the case explicit to whoever comes changing the test later.

```suggestion
			name: "minimal setup",
			params: makeParams(makeParamsArgs{
				blockBytes:  1,
				evidenceAge: 2,
			}),
			valid: true,
		},
		{
			name: "minimal setup 2",
			params: makeParams(makeParamsArgs{
				blockBytes:  1,
				evidenceAge: 2,
				precision:   0 * time.Nanosecond,
 				messageDelay: 0 * time.Nanosecond,
			}),
			valid: true,
		},

```